index.js

```diff
@@ -10,9 +18,35 @@ const init = async () => {
     server.route({
         method: 'GET',
         path: '/',
-        handler: () => {
+        handler: async () => {
+            let client;
+            let result;
+
+            try {
+                client = buildClient();
+                result = await client.listNamespace();
+                const namespaceNames = result.namespace.body.items.map((item) => {
```

Comment:
`listNamespace` allows you to pass a field filter, which would likely make this a lot cleaner.